tests/resources/promise-rejection-events.js

```diff
@@ -446,14 +446,14 @@ async_test(function(t) {
    'all inside a setTimeout');
 
 //
-// Positive unhandledrejection/rejectionhandled tests with delayed attachment
+// Negative unhandledrejection/rejectionhandled tests with task-delayed attachment
 //
 
 async_test(function(t) {
   var e = new Error();
   var p;
 
-  onUnhandledSucceed(t, e, function() { return p; });
+  onUnhandledFail(t, function() { return p; });
```

Comment:
at the point where the task is posted, the about-to-be-notified list is
copied and wiped.

On Fri, Oct 2, 2015 at 1:57 PM Domenic Denicola <notifications@github.com>
wrote:

> In tests/resources/promise-rejection-events.js
> <https://github.com/domenic/unhandled-rejections-browser-spec/pull/13#discussion_r41013947>
> :
>
> >  //
> >
> >  async_test(function(t) {
> >    var e = new Error();
> >    var p;
> >
> > -  onUnhandledSucceed(t, e, function() { return p; });
> > +  onUnhandledFail(t, function() { return p; });
>
> Right, the task already exists, but the list may have been modified since
> the task was posted: "If the about-to-be-notified rejected promises list
> contains promise, remove promise from the about-to-be-notified rejected
> promises list and return."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/domenic/unhandled-rejections-browser-spec/pull/13/files#r41013947>
> .
>